parser/testdata/missing_module.go.mod

```diff
@@ -2,5 +2,5 @@
 go 1.13
 
 require (
-	github.com/mailru/easyjson v0.7.0
+	github.com/cyralinc/easyjson v0.7.10
```

Comment:
Same comment as for parser/testdata/comments_deps.go.mod

Reply:
Same response as above.